study.md

```diff
@@ -25,6 +25,10 @@ In your own words, how Ember "stops breaking the web".
 
 ```md
 <!-- your answer here -->
+Embers router helps us to stop breaking the web.  With ember we get URLs and a working back button.  Ember allows you to
+stop putting your whole application under one URL.  Best practices are essentially built in.  Allows you to manage a large codebase with a many people being apart of
+the team.
```

Comment:
The first parts of this especially. I'm not sure that 

>"Allows you to manage a large codebase with a many people being apart of the team."

is distinct to Ember.